skypy/power_spectrum/tests/test_camb.py

```diff
@@ -36,3 +36,41 @@ def test_camb():
     pzk = camb(wavenumber, redshift, Planck15, 2.e-9, 0.965)
     assert pzk.shape == (len(redshift), len(wavenumber))
     assert allclose(pzk, test_pzk[np.argsort(redshift), :], rtol=1.e-4)
+
+
+@pytest.mark.skipif(CAMB_NOT_FOUND, reason='CAMB not found')
+def test_camb_redshift_zero():
+    """
+    Regression test for #438
+    Test that camb runs succesfully with redshift=0.0
+    """
+    from skypy.power_spectrum import camb
+    redshift = 0.0
+    wavenumber = np.logspace(-4.0, np.log10(2.0), 200)
+    pzk = camb(wavenumber, redshift, Planck15, 2.e-9, 0.965)
+    assert allclose(pzk, test_pzk[0], rtol=1.e-4)
```

Comment:
When running this, I get the following error:
```CAMBError: Error in Fortran called from calc_transfer:
DVERK error```
The rest of tests pass with no error message.

Reply:
This looks like exactly the error reported in #438 that the pull request is supposed to resolve. What version/branch have you installed/checked out when running these tests?

Reply:
You are right. I was working on a condo environment with SkyPy v0.3 for some reason.
It works fine now.